indigo/distribution.yaml

```diff
@@ -11191,11 +11191,19 @@ repositories:
       version: master
     status: maintained
   urdfdom_py:
+    doc:
+      type: git
+      url: https://github.com/ros/urdf_parser_py.git
+      version: 0.3.1
     release:
       tags:
         release: release/indigo/{package}/{version}
       url: https://github.com/ros-gbp/urdfdom_py-release.git
-      version: 0.3.0-2
+      version: 0.3.1-1
+    source:
+      type: git
+      url: https://github.com/ros/urdf_parser_py.git
+      version: 0.3.1
```

Comment:
ok, a tag name is not ok?

Do you suggest I change this manually or re-run bloom somehow?